lms/templates/dashboard/_dashboard_info_language.html

```diff
@@ -1,11 +0,0 @@
-<%! from django.utils.translation import ugettext as _ %>
```

Comment:
That's right. It was included in the `template/dashboard.html` 2 years ago, but later the reference was removed.